drizzlepac/haputils/product.py

```diff
@@ -299,67 +299,102 @@ def add_member(self, edp):
         self.edp_list.append(edp)
 
     def align_to_gaia(self, catalog_name='GAIAedr3', headerlet_filenames=None, output=True,
-                      fit_label='EVM', align_table=None, fitgeom='rscale'):
+                      fit_label='SVM', align_table=None, fitgeom='rscale'):
         """Extract the flt/flc filenames from the exposure product list, as
            well as the corresponding headerlet filenames to use legacy alignment
            routine.
         """
         log.info('Starting alignment to absolute astrometric reference frame {}'.format(catalog_name))
         alignment_pars = self.configobj_pars.get_pars('alignment')
 
-        # Only perform the relative alignment
-        method_name = 'relative'
-
         exposure_filenames = []
         headerlet_filenames = {}
         align_table = None
         crclean = []
 
+        for edp in self.edp_list:
+            exposure_filenames.append(edp.full_filename)
+            headerlet_filenames[edp.full_filename] = edp.headerlet_filename
+            crclean.append(edp.crclean)
+
+        # These should be in a configuration file
```

Comment:
Please review whether or not the 'catalog_list' parameters from the 'run_align' section of the '*/*alignment_all.json' files would be appropriate for these parameters or whether another section might be needed instead.

Reply:
Catalog types and methods have been added to the JSON configuration files in support of the mosaic alignment.

```
`    "run_align":
      {
        "update_hdr_wcs": true,
        "catalog_list": ["GAIAedr3", "GAIADR2", "GAIADR1"],
        "fit_algorithm_list_ngt1": ["match_relative_fit", "match_2dhist_fit", "match_default_fit"],
        "fit_algorithm_list_n1": ["match_2dhist_fit", "match_default_fit"],
        "MIN_CATALOG_THRESHOLD": 3,
        "MIN_OBSERVABLE_THRESHOLD": 10,
        "MAX_FIT_LIMIT": 150,
        "mosaic_catalog_list": ["GAIAedr3", "GAIAdr2", "GAIAdr1" ,"2MASS"],
        "mosaic_fit_list": ["relative", "default"]
      },
`
```